DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/Template_REST_API_AzureFunction_App_Code/Template_REST_API_Function_App_Python/Template_REST_API_Function_App_Python.py

```diff
@@ -36,7 +37,16 @@ def main(mytimer: func.TimerRequest) -> None:
 customer_id = os.environ['workspaceId'] 
 shared_key = os.envviron['workspaceKey']
 log_type = os.envviron['tableName']
-
+logAnalyticsUri = os.environ['logAnalyticsUri']
+if (not str(logAnalyticsUri)): 
```

Comment:
This condition won't work for logAnalyticsUri value None.
Proposed Condition: It will work for None, Empty value and only space values
if ((logAnalyticsUri in (None, '') or str(logAnalyticsUri).isspace())):